data/autoyast_sle12/autoyast_wicked.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <networking>
+    <keep_install_network config:type="boolean">true</keep_install_network>
+  </networking>
+  <bootloader>
+    <device_map config:type="list">
+      <device_map_entry>
+        <firmware>hd0</firmware>
+        <linux>/dev/sda</linux>
+      </device_map_entry>
+    </device_map>
+    <global>
+      <timeout config:type="integer">1</timeout>
+    </global>
+  </bootloader>
+   <scripts>
+    <chroot-scripts config:type="list">
+      <script>
+        <chrooted config:type="boolean">true</chrooted>
+        <interpreter>shell</interpreter>
+        <source>
+          <![CDATA[
+            #!/bin/sh
+            sed -i "s#GRUB_CMDLINE_LINUX_DEFAULT=\"[^\"]*\"#GRUB_CMDLINE_LINUX_DEFAULT=\"video=1024x768-16 plymouth.ignore-serial-consoles console=ttyS0 console=tty splash=verbose showopts\"#" /etc/default/grub

```

Comment:
I have doubts about this line , but I haven't come up with something better . Problem which I see here that actual goal of this line is to **remove**  "resume=/dev/sda1 splash=silent quiet" from GRUB_CMDLINE_LINUX_DEFAULT , but instead of gently erasing what we don't need we just overriding whole variable with sane but still hardcoded values which can cause issues in future. 
Can someone advice sane sed regex for that ?

Reply:
in 2 steps:

    sed -i 's/resume=\/dev\/sda1//' /etc/default/grub
    sed -i 's/splash=silent\ quiet//' /etc/default/grub